docs/src/modules/components/AppTableOfContents.js

```diff
@@ -44,6 +45,11 @@ function getItems(contents) {
 }
 
 const styles = theme => ({
+  '@global': {
+    html: {
+      scrollBehavior: 'smooth',
```

Comment:
@eps1lon What do you think of the scroll behavior when click on TOCs. I kind of slowing me down and making me sick to see the page move. I have to unfocus the page, wait for the end of the animation and focus back the scroll to read the content I'm interested in.

Reply:
I've already removed it. I don't suffer from motion sickness, so I hadn't noticed a problem. If there are n% of users who do, that's a problem.